automated_tests/features/pktvisor.feature

```diff
@@ -1,70 +1,86 @@
-  Then 2 policies must be running
+  Then 4 policies must be running
 
 Scenario: create a policy with dns handler using admin permission
-  Given that a pktvisor instance is running on port default with admin permission
+  Given that a pktvisor instance is running on port available with admin permission
   When create a new policy with dns handler(s)
-  Then 2 policies must be running
+  Then 4 policies must be running
 
 Scenario: create a policy with pcap stats handler using admin permission
-  Given that a pktvisor instance is running on port default with admin permission
+  Given that a pktvisor instance is running on port available with admin permission
   When create a new policy with pcap_stats handler(s)
-  Then 2 policies must be running
+  Then 4 policies must be running
 
 Scenario: delete all policies using admin permission
-  Given that a pktvisor instance is running on port default with admin permission
-  When delete 1 policies
-  Then 0 policies must be running
+  Given that a pktvisor instance is running on port available with admin permission
+  When delete 2 policies
+  Then 2 policies must be running
 
 Scenario: delete 1 policy using admin permission
-  Given that a pktvisor instance is running on port default with admin permission
+  Given that a pktvisor instance is running on port available with admin permission
   When create a new policy with all handler(s)
     And delete 1 policies
-  Then 1 policies must be running
+  Then 2 policies must be running
 
 Scenario: create a policy using user permission
-  Given that a pktvisor instance is running on port 10854 with user permission
+  Given that a pktvisor instance is running on port available with user permission
   When try to create a new policy with all handler(s)
   Then status code returned on response must be 404
-    And 1 policies must be running
+    And 2 policies must be running
 
 
 Scenario: delete 1 policy using user permission
-  Given that a pktvisor instance is running on port default with user permission
+  Given that a pktvisor instance is running on port available with user permission
   When try to delete a policy
   Then status code returned on response must be 404
-    And 1 policies must be running
+    And 2 policies must be running
+
+
+Scenario Outline: pktvisor metrics
+  When run pktvisor instance on port <status_port> with <role> permission
+    And run mocked data <file_name> for this network
+  Then the pktvisor container status must be <pkt_status>
+    And metrics must be correctly generated
+  Examples:
+    | status_port | role | file_name | pkt_status |
+    | available   | user | dns_ipv6_udp.pcap | running    |
+    | available   | user | dns_ipv6_tcp.pcap | running    |
+    | available   | user | dns_ipv4_udp.pcap | running    |
+    | available   | user | dns_ipv4_tcp.pcap | running    |
+    | available   | user | dhcp-flow.pcap | running    |
+    | available   | user | dns_udp_mixed_rcode.pcap | running    |
```

Comment:
Not sure if I understand. The files I saved because talking to Léo, we could assume that what pktvisor is generating today as metrics can be assumed to be correct. If we generate every moment and compare, we will not be able to guarantee that the metrics are correct, right? 'Cause I'll always be comparing a thing to itself